### ISSUE TITLE
PyInstaller3: new error after "merging develop into python3"

### ISSUE THREAD

**Issue by thorstenkampe** [pyinstaller/pyinstaller#1256]
```
23550 INFO: checking EXE
23551 INFO: Building because out00-EXE.toc missing or bad
23552 INFO: Building EXE from out00-EXE.toc
23553 INFO: Onefile Mode - Embedding Manifest into EXE file
23557 INFO: Updating manifest in C:\Users\thorsten\AppData\Local\Temp\tmpejvmlx4m
23557 INFO: Updating resource type 24 name 1 language 0
Traceback (most recent call last):
  File "F:\PortableApps\Python3x\Lib\site-packages\pyinstaller.py", line 18, in <module>
    run()
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\main.py", line 88, in run
    run_build(opts, spec_file, pyi_config)
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\main.py", line 46, in run_build
    PyInstaller.build.main(pyi_config, spec_file, **opts.__dict__)
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\build.py", line 2062, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\build.py", line 2006, in build
    exec(compile(open(spec).read(), spec, 'exec'))
  File "F:\Documents\dev\python\script\script.spec", line 24, in <module>
    console=True , icon='\\PortableApps\\Python3x\\python.exe,0')
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\build.py", line 1312, in __init__
    self.__postinit__()
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\build.py", line 241, in __postinit__
    self.assemble()
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\build.py", line 1389, in assemble
    self.manifest.update_resources(tmpnm, [1]) # 1 for executable
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\utils\win32\winmanifest.py", line 872, in update_resources
    languages)
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\utils\win32\winmanifest.py", line 950, in UpdateManifestResourcesFromXML
    languages or [0, "*"])
  File "F:\PortableApps\Python3x\Lib\site-packages\PyInstaller\utils\win32\winresource.py", line 205, in UpdateResources
    win32api.UpdateResource(hdst, type_, name, data, language)
TypeError: 'str' does not support the buffer interface
```

**Comment by thorstenkampe** [comment]
Additional information: the error happens only with `--onefile`...

**Comment by htgoebel** [comment]
Thanks for the report. I don't have any Windows system, so I can not track this down. Did thiswork just before the merge? Please try `git bisect`-ing it. A pull-request would be great.

**Comment by thorstenkampe** [comment]
yes, it worked not long ago - before the merge. About the git stuff: I downloaded the zip fill to avoid git.

**Comment by chfoo** [comment]
I tried to git bisect as much as I could:
```
git bisect start
# bad: [ce3fc62800766a1acc73ff636e81c6e1e275f1d3] Merge branch 'develop' into python3
git bisect bad ce3fc62800766a1acc73ff636e81c6e1e275f1d3
# good: [dc4adc6639475a3e8ce1418d1dd57cb91076380f] Merge pull request #1159 from matysek/python3
git bisect good dc4adc6639475a3e8ce1418d1dd57cb91076380f
# bad: [e5ffdc065cc02d671a9376e905a7ee6b4d93af80] Merge branch 'tallforasmurf-python3' into python3.
git bisect bad e5ffdc065cc02d671a9376e905a7ee6b4d93af80
# good: [2b0ec9dc2ab39fd46f0cb463379f8d6ec6fb75d4] Merge pull request #1174 from matysek/python3
git bisect good 2b0ec9dc2ab39fd46f0cb463379f8d6ec6fb75d4
# good: [72833cde9fe205e941142cafaace6834554ec7b5] Merge pull request #1194 from leycec/python3
git bisect good 72833cde9fe205e941142cafaace6834554ec7b5
# bad: [1474d59c20a6fa1504d17f1fc43c6d91c8f4bdd2] Merge branch 'python3-merge' into python3.
git bisect bad 1474d59c20a6fa1504d17f1fc43c6d91c8f4bdd2
# bad: [e886b451ffeae9b5bc19d7a123baacf76ac7b92e] Merge branch 'develop' into python3.
git bisect bad e886b451ffeae9b5bc19d7a123baacf76ac7b92e
```

**Comment by codewarrior0** [comment]
This is fixed by #1259
